homework.js

```diff
@@ -3,24 +3,54 @@
 // assign your variable the value 6
 // console log your variable
 
+let fruit = 6;
+console.log(fruit);
 
 // write a function called product that accepts two numbers and 
 // logs the product in the console and then
 // returns the answer when you multiply them both together
 // call that function on the numbers 4 & 689 to test it
 
-// Problem 2
-// write a function called evens that takes an array of numbers, removes 
-// all the numbers that are odd and returns a new array with only the even numbers
-// remember that even numbers have a remainder of 0 when you divide them by 2
-// call your function on this array of numbers [4, 37, 18, 29, 52, 103, 22]
+function product (numberOne, numberTwo){
+  let product = numberOne * numberTwo;
+  console.log(product);
+}
+(4, 689);
+
+//Problem 2
+//write a function called evens that takes //an array of numbers, removes 
+//all the numbers that are odd and returns //a new array with only the even numbers
+//remember that even numbers have a //remainder of 0 when you divide them by 2
+//call your function on this array of numbers [4, 37, 18, 29, 52, 103, 22]
+
+let arr = [4, 37, 18, 29, 52, 103, 22];
+
+function evens (a) {
+  let newEvens = [];{
+    for
+  }
+
+  return newEvens;
+}
+
 
 //Problem 3
 // write a function called five that takes a word and prints it in the console
 // five times
 // call your function on the word "pineapple" to test it
 
+function five (string, number){
+  let word = "pineapple";
+  console.log(word.repeat(5));
+}
 //Problem 4
 // write a function called shortWords that takes an array of words and then
 // returns a new array with only the words that are 3 letters long or less
 // call your function on this array ["basin", "pit", "cheese", "bun", "river"]
+
+function shortWords (arr){
+let newArray = [];
+let originalArray = ["basin", "pit", "cheese", "bun", "river"];
+for originalArray.length(if i <=3) 
```

Comment:
You'll want to check the length of each word, not the array. 
`for (let word of arr)` Your argument name needs to remain consistent throughout the function. If you define a specific array to call the function on, do that outside the function.